package.json

```diff
@@ -23,26 +23,37 @@
     "url": "https://github.com/apache/daffodil-vscode/issues"
   },
   "scripts": {
+    "omega-edit-download": "tsc -p ./ && node -e 'require(\"./out/omega_edit/download.js\").downloadServer()'",
     "vscode:prepublish": "yarn run package-ext",
     "precompile": "node -p \"'export const LIB_VERSION = ' + JSON.stringify(require('./package.json').version) + ';'\" > src/version.ts",
-    "compile": "tsc -p ./",
+    "compile": "tsc -p ./ && node out/omega_edit/download.js",
```

Comment:
I removed the run-script-os stuff (see below comment), and when I run `yarn build` it never downloads the omega zip file. Should this have the `require` and `downloadServer` stuff mentioned above? Or should something else depend on oemga-edit-download?

Reply:
This may be because of the removing of the run-script-os stuff as that is required to get the proper pre and post build items to run for the OS you are on if you see my above comment you should be able to fix your issue with run-script-os

Reply:
This did not fix the download as far as I can tell. I don't see the zip file anywhere in the repo or in the .vsix file.

Reply:
It doesn't exist in `src/omega_edit/omega-edit-scala-server-0.9.8.zip`? That is weird if it isn't there as it was working for me I will try to dig around see if I can find what could cause the issue

Reply:
Correct, I ran `git clean -xdf && yarn install && yarn build` and it's not there.

I do see this line in the output
```
$ tsc -p ./ && node out/omega_edit/download.js
```
But it doesn't seem to download anything.

Reply:
Uh that is weird I will look into that. Also I will try to fix the windows CI, they seem to timeout on the `yarn build` for some reason I will have to try to figure out

Reply:
@stevedlawrence Please pull down the latest and try to download again it should work now. It was a difference between `'` and `"` in the inline script to download the server that was causing the error

Reply:
I also have no idea what is going on with the Windows CI, I tried reverting back to previous version of sbt but it didn't work so I am not really sure why those are timing out on this branch